crates/ruff_linter/src/rules/airflow/rules/removal_in_3.rs

```diff
@@ -646,6 +653,68 @@ fn removed_name(checker: &mut Checker, expr: &Expr, ranged: impl Ranged) {
     }
 }
 
+fn extract_name_from_slice(slice: &Expr) -> Option<String> {
+    if let Expr::StringLiteral(ExprStringLiteral { value, .. }) = slice {
+        Some(value.to_string())
+    } else {
+        None
+    }
+}
```

Comment:
I think we can do better with `?` but I’m not fluent in Rust enough to make a concrete suggestion…

Reply:
Changed it: https://github.com/astral-sh/ruff/pull/15144/commits/72dcad72e3f0ea0e6c008341bf5f6ec615d7fdb4